polkadot/node/subsystem-util/src/reputation.rs

```diff
@@ -75,6 +76,10 @@ impl ReputationAggregator {
 		peer_id: PeerId,
 		rep: UnifiedReputationChange,
 	) {
+		if rep.cost_or_benefit() < 0 {
+			gum::debug!(target: LOG_TARGET, peer = ?peer_id, ?rep, "Modify reputation");
+		}
```

Comment:
nit: since only printing this on reputation loss:
```suggestion
		if rep.cost_or_benefit() < 0 {
			gum::debug!(target: LOG_TARGET, peer = ?peer_id, ?rep, "Reduce reputation");
		}
```